modules/discoveryBidAdapter.js

```diff
@@ -55,24 +58,21 @@ const NATIVERET = {
 };
 
 /**
- * 获取用户id
+ * 获取并生成用户的id
  * @return {string}
  */
-const getUserID = () => {
-  let idd = storage.getCookie(COOKIE_KEY_SSPPID);
-  let idm = storage.getCookie(COOKIE_KEY_MGUID);
-
-  if (idd && !idm) {
-    idm = idd;
-  } else if (idm && !idd) {
-    idd = idm;
-  } else if (!idd && !idm) {
-    const uuid = utils.generateUUID();
-    storage.setCookie(COOKIE_KEY_MGUID, uuid);
-    storage.setCookie(COOKIE_KEY_SSPPID, uuid);
-    return uuid;
+export const getPmgUID = () => {
+  let pmgUid = storage.getCookie(COOKIE_KEY_PMGUID);
+
+  if (!pmgUid) {
+    pmgUid = utils.generateUUID();
+    const date = new Date();
+    date.setTime(date.getTime() + COOKIE_RETENTION_TIME);
+    try {
+      storage.setCookie(COOKIE_KEY_PMGUID, pmgUid, date.toUTCString());
```

Comment:
Setting a User Id cookie should be moved to the getUserSyncs function and run only when the publisher configures to allow cookies. This function should only retrieve the cookie if it exists.

Reply:
Hi, i have change it, please review again.Thanks.